backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,28 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../architecture/admission-plug-ins.adoc[Admission plug-ins]
+* xref:../../architecture/admission-plug-ins.adoc#admission-webhooks-about_admission-plug-ins[Webhook admission plug-ins]
+* xref:../../architecture/admission-plug-ins.adoc#admission-webhook-types_admission-plug-ins[Types of webhook admission plug-ins]
+
+[id="issues-with-velero-and-admission-workbooks"]
+== Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore. If you have workloads with admission webhooks, you might need to use an additional Velero plug-in or make changes to how you restore the workload..
+
+Typically, workloads with admission webhooks require you to create a resource of a specific kind first.
+
```

Comment:
@kaovilai  Can you give some information about how the creation of resources might help with the restore.

Reply:
If you restore/create the top level object first [like service.serving.knative.dev](https://github.com/knative/serving/issues/10926#issuecomment-886434177) then the child resources are typically created already and so it's more likely that admission webhook problems are resolved because child resources (typically blocked by these admission webhooks) do not need to be created by velero